### DISCUSSION TITLE
Widget: More information in Pie-Chart

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#407] [Suggestions]
I just thought about having more information in the pie-chart. When you select: 
"new characters" it's currently showing how many % of the total are from server x. It would be nice, so see whats the total is and maybe how many players are including the %. 

![image](https://github.com/gameserverapp/Platform/assets/99832504/3de5b79b-f6e7-4f52-aa58-146e27b63752)

Old text:

More information for dashboard widgets like the one in following picture

Player per Server
How many players are 28%

**Comment by mvaessen** [comment]
Hi, could you please make the title more descriptive about what your suggestion is about.

**Comment by LimitlessDrag0n** [comment]
Okay, sorry about that.

I would like to see, how many players are on all servers. Maybe as a field besides the name of the server by the legend, how many are there per server while hovering over one server  or a tool tip how many those 28% are.
